src/content/blog/astro-tutorial.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Astro Tutorial
+description: 'Wrapping up Astro Docs week with our biggest Docs announcement to date: our first Astro Tutorial.'
+publishDate: 'November 4, 2022'
+socialImage: '/assets/blog/astro-tutorial/cover.png'
+coverImage: '/assets/blog/astro-tutorial/cover.png'
+lang: 'en'
+authors:
+    - chris
+    - sarah
```

Comment:
These should definitely be flipped to show Sarah first!

```suggestion
    - sarah
    - chris
```

Reply:
awww....